app/controllers/exercises_controller.rb

```diff
@@ -9,7 +9,13 @@ class ExercisesController < ApplicationController
   before_action :start!, only: :show
 
   def show
+    @embedded_mode = params[:embed] == 'true' && Organization.current.settings.embeddable?
```

Comment:
As you delegate the method to settings in platform, you could just send embeddable? to the organization